scripts/globals/spells/trust/volker.lua

```diff
@@ -11,3 +15,20 @@ end
 function onSpellCast(caster, target, spell)
     return tpz.trust.spawn(caster, spell)
 end
+
+function onMobSpawn(mob)
+    mob:addSimpleGambit(ai.t.MASTER, ai.c.HPP_LT, 50,
+                        ai.r.JA, ai.s.SPECIFIC, tpz.ja.PROVOKE)
+
+    mob:addSimpleGambit(ai.t.SELF, ai.c.NOT_STATUS, tpz.effect.BERSERK,
+                        ai.r.JA, ai.s.SPECIFIC, tpz.ja.BERSERK)
```

Comment:
According to bgwiki, Volker uses a weaponskill which gives beserk, not the actual JA